src/main/java/META-INF/spring/app-config.xml

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <beans xmlns="http://www.springframework.org/schema/beans"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:context="http://www.springframework.org/schema/context"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:context="http://www.springframework.org/schema/context" xmlns:util="http://www.springframework.org/schema/util"
 	xsi:schemaLocation="http://www.springframework.org/schema/beans
 							http://www.springframework.org/schema/beans/spring-beans-3.1.xsd
 							http://www.springframework.org/schema/context
-							http://www.springframework.org/schema/context/spring-context-3.1.xsd">
+							http://www.springframework.org/schema/context/spring-context-3.1.xsd
+							http://www.springframework.org/schema/util
+							http://www.springframework.org/schema/util/spring-util-3.1.xsd">
```

Comment:
@tarelli [This one](https://github.com/openworm/org.geppetto.simulator.external/blob/b6d865170587f438bb14329e4623ba244bf161de/src/main/java/META-INF/spring/app-config.xml#L45-L48) Really nice actually. It ends up being a hashmap. Really useful as every simulator can have some specific parameters (as for instance the appId and appName).